Packs/SymantecDLP/Integrations/SymantecDLPV2/SymantecDLPV2.py

```diff
@@ -560,21 +695,33 @@ def list_incidents_command(client: Client, args: Dict[str, Any]) -> CommandResul
     limit = arg_to_number(args.get('limit', 50))
     page = arg_to_number(args.get('page', 1))
     page_size = arg_to_number(args.get('page_size'))
-
-    incidents_result = client.get_incidents_request(creation_date, status_ids, severities_dlp, incident_types_dlp,
-                                                    limit * page)  # type: ignore
+    raw_filter = args.get('raw_filter')
+    try:
+        incidents_result = client.get_incidents_request(creation_date, status_ids, severities_dlp, incident_types_dlp,
+                                                        limit * page, None, raw_filter)  # type: ignore
+    except DemistoException as e:
+        if '400' in str(e):
+            return_error = 'Error 400: Bad filter. Ensure the filter meets the requirements shown at https://apidocs.securitycloud.symantec.com/#/doc?id=incidentlists.'
+        else:
+            raise e
```

Comment:
```suggestion
        incidents_result = client.get_incidents_request(creation_date, status_ids, severities_dlp, incident_types_dlp, limit * page, raw_filter=raw_filter)  # type: ignore

```

Reply:
Fixed

Reply:
it doesn't seem like it has been fixed, can you please refer?